aws_securityhub/help.md

```diff
@@ -195,6 +193,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 1.0.2 - Hub styling update
```

Comment:
Don't need to bump

Reply:
`* 1.4.0 - New trigger Risk Detection`  you sure?